src/plugins/intel_npu/src/al/include/intel_npu/npuw_private_properties.hpp

```diff
@@ -412,6 +412,14 @@ static constexpr ov::Property<uint32_t> max_prompt_len{"NPUW_LLM_MAX_PROMPT_LEN"
  */
 static constexpr ov::Property<uint32_t> min_response_len{"NPUW_LLM_MIN_RESPONSE_LEN"};
 
+/**
+ * @brief
+ * Type: ov::AnyMap.
+ * Tell NPUW the configuration for compilation of prefill model.
+ * NOTE: !! Write-only !!
+ */
+static constexpr ov::Property<ov::AnyMap> prefill_config{"NPUW_LLM_PREFILL_CONFIG"};
```

Comment:
Because we pass them as `properties`.

Reply:
Yes, this is the exact question. My point was not to make it as `property`.

It will only simplify implementation, don't see use case where user would like to query these `properties` for read/write, is there?

Reply:
Discussed internally